web/package.json

```diff
@@ -143,7 +143,7 @@
     "history": "^4.9.0",
     "js-yaml": "3.14.0",
     "lodash": "4.17.21",
-    "markdown-it": "^10.0.0",
+    "markdown-it": "^12.3.2",
```

Comment:
seems fine- i don't see any breaking changes from 10 to 12 in the changelog: https://github.com/markdown-it/markdown-it/blob/master/CHANGELOG.md